lib/views/frontend/spree/orders/payment/_paypal_commerce_platform.html.erb

```diff
@@ -1,3 +1,8 @@
+<script>
```

Comment:
this file already has `Spree.current_order_id` and token being set in the javascript [here](https://github.com/solidusio-contrib/solidus_paypal_commerce_platform/blob/610c00ec8414408ca73f1225ef1bc18d71eddc56/lib/views/frontend/spree/orders/payment/_paypal_commerce_platform.html.erb#L8) - a couple of lines below here - so we can safely leave this file alone - I asked you to add it there without checking if it was there in the first place.

Reply:
No problem at all, I'll just rebase and remove that second commit adding it. :)